rustls/src/verify.rs

```diff
@@ -569,12 +589,21 @@ impl ClientCertVerifier for AllowAnyAuthenticatedClient {
         now: SystemTime,
     ) -> Result<ClientCertVerified, Error> {
         let (cert, chain, trustroots) = prepare(end_entity, intermediates, &self.roots)?;
+        let crls = self
+            .crls
+            .iter()
+            .map(|crl| crl.parse())
+            .collect::<Result<Vec<_>, _>>()?;
```

Comment:
leaving a note we could `unwrap()` here now that we're pre-validating the CRL DER in `new`.